Makefile

```diff
@@ -5,7 +5,7 @@ MINI_LAB_KUBECONFIG := $(shell pwd)/../mini-lab/.kubeconfig
 
 include $(COMMONDIR)/Makefile.inc
 
-release:: protoc spec check-diff all ;
```

Comment:
Without the protoc target the `check-diff` target will never find changes.

Reply:
This is done in CI
https://github.com/metal-stack/metal-api/pull/279/files#diff-023e805b335fe6fb703da891ae5add592e98632ab791a1bbcf2c67a34ecfdc67R27